specs/README.md

```diff
@@ -7,11 +7,12 @@ For smaller architectural changes, opening an issue and gathering consensus on a
 The process to create and approve a spec is:
 
 1. In a new branch, create a copy of one of the following spec templates as described below:
-   - if proposing other architecture or infrastructure changes, use the [high-level design template](/specs/templates/high-level-design.md) and place the spec at `/specs/title-of-your-proposal/README.md`.
-   - if proposing a new component, use one of the following templates and place the spec at `packages/nimble-components/src/component-name/specs/README.md`:
-      - use the [FAST-based component template](/specs/templates/fast-based-component.md) if leveraging a FAST component.
-      - use the [custom component template](/specs/templates/custom-component.md) if building a new component from scratch or on top of a different library.
-   - if proposing extra component-specific effort/features use the [high-level design template](/specs/templates/high-level-design.md), and place the spec at `packages/nimble-components/src/component-name/specs/topic.md`
+   - If proposing other architecture or infrastructure changes, use the [high-level design template](/specs/templates/high-level-design.md) and place the spec at `/specs/title-of-your-proposal/README.md`.
```

Comment:
While not new with your change, I think the word "other" in this line is unnecessary/confusing. I think this should just be

```suggestion
   - If proposing architecture or infrastructure changes, use the [high-level design template](/specs/templates/high-level-design.md) and place the spec at `/specs/title-of-your-proposal/README.md`.
```